graphql/execute.lua

```diff
@@ -278,14 +278,48 @@ local function getFieldEntry(objectType, object, fields, context)
       if argument and argument.value then
           positions[pos] = {
               name=argument.name.value,
-              value=arguments[argument.name.value]
+              value=arguments[argument.name.value],
```

Comment:
This change is not related to your patch.

Reply:
Resolved